contrib/tablefunc/data/connectby_int.data

```diff
@@ -0,0 +1,9 @@
+1	\N
```

Comment:
why not use simple insert?

Reply:
Directory data was copied from the [upstream/master](https://github.com/greenplum-db/gpdb/tree/master/contrib/tablefunc/data), also this directory exists at [postgres](https://github.com/postgres/postgres/tree/master/contrib/tablefunc/data). I think we have to follow this approach (to prevent changes in sql test file), to prevent further merge conflicts.